### DISCUSSION TITLE
Hey! Is it possible to embed filament tables to custom views? + with ability to edit records

### DISCUSSION THREAD

**Discussion by init-imangali** [filamentphp/filament#13381] [Help]
### Package

Table builder

### Package Version

v3.0.0

### How can we help you?

Hello guys!

In short i need to make table with ability to editing of some fields if user has certain roles. Of course admins can edit all fields. As example orders table. But giving full admin panel access to some workers that must just upload some images or track codes is... visually wrong when they has only one resource to a whole admin page.

So on the other hand pages under /dashboard available to all users and there just basic info like user profile, edit profile, my orders etc. so here somewhere if i put /dashboard/orders, which has default site header, footer, and sidebar with user avatar and list of links for dashboard pages under it. With highlighted active link. All of it maded with pure blade components. And on the right side there is quite a lot space for content. Where i wanted to embed some table. and further in /dashboard/orders/:id/edit at the same content area make form to edit this order.

So im curious, should i make my own bycicle or is it possible to reuse filament for that?

I'll be really thankfull if you answer to this if you know smth bout it mate!

**Comment by leandrocfe** [comment]
You could create a livewire component and add a table
https://filamentphp.com/docs/3.x/tables/adding-a-table-to-a-livewire-component#adding-the-table
